doc/whats_new/v0.20.rst

```diff
@@ -103,6 +103,14 @@ Changelog
   :class:`decomposition.IncrementalPCA` when using float32 datasets.
   :issue:`12338` by :user:`bauks <bauks>`.
 
+Miscellaneous
+.............
+
+- |Fix| Make sure to avoid raising ``FutureWarning`` when calling
```

Comment:
Seems that FutureWarning is only a proposal (not implemented) in numpy?

Reply:
It's pretty likely to be implemented. If not we can always update our changelog later.

Reply:
Could you give me some pointers about where the proposal/discussion about `FutureWarning` and `VisibleDeprecationWarning` in `numpy` is happening? 

It'll effect the semantics behind `sklearn.utils.testing.assert_no_warnings`: https://github.com/scikit-learn/scikit-learn/pull/12478/commits/d6972a9731a024bd4367bd703e4d3be26018d9ee

Reply:
see https://github.com/numpy/numpy/issues/12263 mentioned above.

Reply:
The FutureWarning PR has been merged into numpy master as well: https://github.com/numpy/numpy/pull/12280/files